### PR TITLE
60-Add-2022-Survey-Questions

### LINKED ISSUE #60: Add 2022 Survey Questions
Add the questions used in 2022 SOD surevey

### PR DESCRIPTION
**Pull request by davidstanke**
Fixes #60

### CODE REVIEW

**Comment by davidstanke** [comment]
Preview this PR at https://draft-dora-dev--pr182-cskb2rue.web.app

**Comment by davidstanke** [comment]
Preview this PR at https://draft-dora-dev--pr182-cskb2rue.web.app

File: hugo/data/survey_questions.json [hunk 1]
```diff
@@ -1,20 +1,311 @@
...
+                            "My team prioritizes keeping the software deployable over working on new features"
+                        ]
+                    }
+                ]
+            },
+            {
+                "heading": "Software Delivery Performance",
+                "question_groups": [
+                    {
+                        "questions": [
```
Comment (nathenharvey) [inline hugo/data/survey_questions.json:78]: should the options be listed?

**Review by nathenharvey — APPROVED** [review]
I approve these changes but have left some additional comments mainly focused on clarifying what options were presented to survey participants.  These are not blockers and I would be happy to see those discussed in an issue or pull request following these changes being merged.

File: hugo/data/survey_questions.json [hunk 2]
```diff
@@ -1,20 +1,311 @@
...
+                            "It has well-defined targets for reliability (such as Service Level Agreements / Service Level Objectives) that are clearly communicated among the team and to customers",
+                            "My team regularly reviews and revises reliability targets based on evidence",
+                            "When we miss our reliability targets, we perform improvement work and/or re-prioritize",
+                            "I know what its reliability actually was in the most recent period",
+                            "Reliability reviews are performed throughout the development process for all major features on the applications I work on"
+                        ]
+                    },
+                    {
+                        "questions": [
+                            "How often do your services meet reliability expectations (i.e., SLOs)?"
```
[the remark this answers is not in the document]
Comment (davidstanke) [reply hugo/data/survey_questions.json:228]: I think it was a slider... yeah, it doesn't belong to the "these next questions" header. I opened #183 to clarify

File: hugo/data/survey_questions.json (hunk 1 above)
Comment (davidstanke) [reply hugo/data/survey_questions.json:78]: Good call! Added #184

File: hugo/data/survey_questions.json [hunk 3]
```diff
@@ -1,20 +1,311 @@
...
+                            "The cloud my product or service runs on serves multiple teams and applications, with compute and infrastructure resources dynamically assigned and re-assigned based on demand"
+                        ]
+                    }
+                ]
+            },
+            {
+                "heading": "Organizational Performance",
+                "question_groups": [
+                    {
+                        "description": "Select the option that indicates how your organization performed for the following goals over the past year.",
```
[the remark this answers is not in the document]
Comment (davidstanke) [reply hugo/data/survey_questions.json:157]: Yes it would seem so! I'll track that down and add it -- opened #185 to track

**Pull request merged**